src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/prisonapi/LegacyNonAssociationDetails.kt

```diff
@@ -58,13 +58,11 @@ data class LegacyOffenderNonAssociation(
   @Schema(description = "Last name", required = true, example = "Bloggs")
   val lastName: String,
   @Schema(description = "Reason code for the non-association", required = true, example = "PER")
-  val reasonCode: String,
+  val reasonCode: LegacyReason,
   @Schema(description = "Reason for the non-association", required = true, example = "Perpetrator")
   val reasonDescription: String,
   @Schema(description = "Description of the agency (e.g. prison) the offender is assigned to", required = true, example = "Moorland (HMP & YOI)")
   val agencyDescription: String,
   @Schema(description = "Description of living unit (e.g. cell) the offender is assigned to.", required = false, example = "MDI-2-3-4")
   val assignedLivingUnitDescription: String?,
-  @Schema(description = "ID of living unit (e.g. cell) the offender is assigned to.", required = false, example = "234")
-  val assignedLivingUnitId: Long? = null,
```

Comment:
are you sure you wanna drop this? it appears in
https://api.prison.service.justice.gov.uk/swagger-ui/index.html#/offenders/getNonAssociationDetails

Reply:
Yes but its not used anywhere